keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,562 @@
+        top_filters = round_filters(
+            filters=1280,
+            width_coefficient=width_coefficient,
+            min_depth=min_depth,
+            depth_divisor=depth_divisor,
+        )
+
+        x = layers.Conv2D(
+            filters=top_filters,
+            kernel_size=1,
+            strides=1,
+            kernel_initializer=conv_kernel_initializer(),
+            padding="same",
+            data_format="channels_last",
+            use_bias=False,
+            name="top_conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=BN_AXIS,
+            momentum=0.9,
+            name="top_bn",
+        )(x)
+        x = layers.Activation(activation=activation, name="top_activation")(x)
+
+        pyramid_level_inputs[pyramid_level_tracker] = x.node.layer.name
+
+        inputs = img_input
+
+        # Create model.
+        super().__init__(inputs=inputs, outputs=x, **kwargs)
+
+        self.include_rescaling = include_rescaling
+        self.width_coefficient = width_coefficient
+        self.depth_coefficient = depth_coefficient
+        self.default_size = default_size
+        self.dropout_rate = dropout_rate
+        self.drop_connect_rate = drop_connect_rate
+        self.depth_divisor = depth_divisor
+        self.min_depth = min_depth
+        self.activation = activation
+        self.block_args = input_block_args
+        self.input_tensor = input_tensor
+        self.pyramid_level_inputs = pyramid_level_inputs
+
+    def get_config(self):
+        config = super().get_config()
+        config.update(
+            {
+                "include_rescaling": self.include_rescaling,
+                "width_coefficient": self.width_coefficient,
+                "depth_coefficient": self.depth_coefficient,
+                "default_size": self.default_size,
+                "dropout_rate": self.dropout_rate,
+                "drop_connect_rate": self.drop_connect_rate,
+                "depth_divisor": self.depth_divisor,
+                "min_depth": self.min_depth,
+                "activation": self.activation,
+                "block_args": self.block_args,
+                # Remove batch dimension from `input_shape`
+                "input_shape": self.input_shape[1:],
+                "input_tensor": self.input_tensor,
+            }
+        )
+        return config
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return copy.deepcopy(backbone_presets)
+
+    @classproperty
+    def presets_with_weights(cls):
+        """Dictionary of preset names and configurations that include
+        weights."""
+        return copy.deepcopy(backbone_presets_with_weights)
+
+
+BASE_DOCSTRING = """Instantiates the {name} architecture.
+
+    Reference:
+    - [EfficientNetV2: Smaller Models and Faster Training](https://arxiv.org/abs/2104.00298)
+      (ICML 2021)
+
+    This function returns a Keras image classification model.
```

Comment:
This is no longer true (same for the line(s) below)

Reply:
Good point - done.